enhance: “つながりの公開範囲”がフォロー・フォロワー個別設定できるように

zawa-ch opened the pull request:
## What
このPRは「つながりの公開範囲」をフォロー/フォロワーで個別に設定できるようになります。

## Why
このPRは #12072 を解決することができます。 (Resolve #12072)

## Additional info (optional)
![Screenshot 1](https://github.com/misskey-dev/misskey/assets/23190298/fb1b8056-f8cc-48d4-aeb4-d915b7d7ad93)
![Screenshot 2](https://github.com/misskey-dev/misskey/assets/23190298/1d92a0b1-26f5-429d-be2d-f5d24fe4866d)


## Checklist
- [x] Read the [contribution guide](https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md)
- [x] Test working in a local environment
- [ ] (If needed) Add story of storybook
- [x] (If needed) Update CHANGELOG.md
- [x] (If possible) Add tests

zawa-ch committed:
Enhance: “つながりの公開範囲”がフォロー・フォロワー個別設定できるように (#12072)

Commit:
Merge branch 'develop' into develop-ffvisibility

syuilo commented:
- ja-JP以外の翻訳ファイルの編集はcrowdinとコンフリクトしてしまうので避けていただけると🙏
- followerVisibility -> followersVisibility かも

zawa-ch commented:
> * ja-JP以外の翻訳ファイルの編集はcrowdinとコンフリクトしてしまうので避けていただけると🙏

わかりました、この部分は差し戻します

> * followerVisibility -> followersVisibility かも

ア！

zawa-ch committed:
refactor: crowdin 編集部分のコミットを打ち消し

https://github.com/misskey-dev/misskey/pull/12702#issuecomment-1859417158

zawa-ch committed:
refactor: オブジェクトの名前修正

https://github.com/misskey-dev/misskey/pull/12702#issuecomment-1859417158

zawa-ch commented:
> > * ja-JP以外の翻訳ファイルの編集はcrowdinとコンフリクトしてしまうので避けていただけると🙏
> 
> わかりました、この部分は差し戻します
> 
> > * followerVisibility -> followersVisibility かも
> 
> ア！

修正しました

zawa-ch committed:
fix: 設定項目の説明を削除

名称が具体的になって必要なくなったため
https://github.com/misskey-dev/misskey/pull/12702#discussion_r1429932463

zawa-ch commented on locales/ja-JP.yml:
```diff
@@ -881,8 +881,10 @@ makeReactionsPublicDescription: "あなたがしたリアクション一覧を
-ffVisibility: "つながりの公開範囲"
-ffVisibilityDescription: "自分のフォロー/フォロワー情報の公開範囲を設定できます。"
+followingVisibility: "フォローの公開範囲"
+followingVisibilityDescription: "自分のフォロー情報の公開範囲を設定できます。"
```
そうですね、消しちゃいましょう

zawa-ch commented:
![Screenshot](https://github.com/misskey-dev/misskey/assets/23190298/07390149-c06d-410a-ae60-7124adbfa027)

syuilo commented:
👍👍
